### PR TITLE
Require first app before making configurable report

### PR DESCRIPTION
**Pull request by nickpell**
Before:

screen shot 2015-03-10 at 6 35 55 pm

After:

screen shot 2015-03-10 at 6 35 32 pm

@NoahCarnahan @czue

### CODE REVIEW

**Comment by NoahCarnahan** [comment]
looks good. Thanks for the screen shots.

File: corehq/apps/userreports/templates/userreports/create_new_report_builder.html [hunk 1]
```diff
@@ -13,3 +13,16 @@
...
     </script>
 {% endblock %}
+
+{% block main_column %}
+    {% if sources_map %}
+        {{ block.super }}
+    {% else %}
+        <div style="padding: 1em; text-align: center">
+            <p style="font-size: 1.5em">
+                Before creating a new report, you need an application.
```
Comment (czue) [inline corehq/apps/userreports/templates/userreports/create_new_report_builder.html]: (missing translation)

**Comment by czue** [comment]
good call, thanks!

File: corehq/apps/userreports/templates/userreports/create_new_report_builder.html (hunk 1 above)
Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/create_new_report_builder.html]: oops: https://github.com/dimagi/commcare-hq/pull/5840

**Pull request merged**